### PR TITLE
Update README.md for local dev instructions

### PR DESCRIPTION
**Pull request by lukechurch**
@devoncarew PTAL - I'm not sure the way I'm suggesting people use grind is correct, I've had some problems running the travis script locally. Could you correct as appropriate?

### CODE REVIEW

**Comment by lukechurch** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -23,6 +23,19 @@ work in progress.
 Please file reports on the
 [GitHub Issue Tracker](https://github.com/dart-lang/dart-pad/issues).
 
+## Running locally
```
Comment (devoncarew) [inline README.md]: This isn't necessary to run the front-end; you can launch web/index.html (the desktop version) or web/mobile.html (the mobile version) directly from the editor.

I think these instructions may confuse people a bit. Perhaps add a disclaimer that the front-end can be run w/o any server, but if you want to test the auto-redirect, here are some steps?

File: README.md [hunk 2]
```diff
@@ -23,6 +23,19 @@ work in progress.
...
+
+The project contains a tiny appengine for redirecting to gists and the mobile UI. To run this:
+- Download the AppEngine SDK for Python here: https://cloud.google.com/appengine/downloads
+- Navigate to the root of the checkout folder
+- run `pub get`
```
Comment (devoncarew) [inline README.md]: you can remove this line

File: README.md [hunk 3]
```diff
@@ -23,6 +23,19 @@ work in progress.
...
+- run `dart tool/grind.dart build`
```
Comment (devoncarew) [inline README.md]: change to "run `grind build`"

**Comment by devoncarew** [comment]
lgtm w/ comments

Please make yourself right with @googlebot

File: README.md (hunk 1 above)
Comment (lukechurch) [inline README.md]: Yup, sg, will do.

File: README.md (hunk 2 above)
Comment (lukechurch) [inline README.md]: It doesn't seem to do the right thing if I do. I ran into this earlier this evening. Lets debug next week.

File: README.md (hunk 3 above)
Comment (lukechurch) [inline README.md]: This also doesn't work on my local machine. I think at the very least we need to either include a link to the grinder install setup instructions, or to inline them here.

**Comment by lukechurch** [comment]
Thanks @devoncarew .

Issues with @googlebot seem to be related to using GitHub's UI for modification, I'll route things through a local change in future.

File: README.md (hunk 3 above)
Comment (stevemessick) [inline README.md]: Before that you need to do: sudo npm install -g bower
Which means you may need to install node.js first.
Running the command Luke used, I got this at the end:

  build/web/mobile.html original: 6k
  vulcanize --strip --output mobile.html mobile.html

ProcessException: No such file or directory
  Command: vulcanize --strip --output mobile.html mobile.html
#0      _ProcessImpl._runAndWait (dart:io-patch/process_patch.dart:474)
#1      _runNonInteractiveProcessSync (dart:io-patch/process_patch.dart:626)
#2      Process.runSync (dart:io-patch/process_patch.dart:64)
#3      run (package:grinder/src/run.dart:26:34)
#4      build (file:///Users/messick/src/editors/dart-pad-master/tool/grind.dart:62:3)

Comment (devoncarew) [inline README.md]: OK, this makes sense now. The bot is running this script: https://github.com/dart-lang/dart-pad/blob/master/tool/travis.sh

Which is setting up bower and vulcanize (and npm/node is already available).

**Comment by devoncarew** [comment]
still relevant?

**Comment by lukechurch** [comment]
@devoncarew Lets chat about this in a couple of weeks and review. I still have a bunch of problems with my local build setup seemingly.

**Pull request closed without merging**